var/spack/repos/builtin/packages/py-triton/package.py

```diff
@@ -11,14 +11,21 @@ class PyTriton(PythonPackage):
 
     homepage = "https://github.com/openai/triton"
     url = "https://github.com/openai/triton/archive/refs/tags/v2.1.0.tar.gz"
+    git = "https://github.com/openai/triton.git"
 
     license("MIT")
 
+    version("main", branch="main")
     version("2.1.0", sha256="4338ca0e80a059aec2671f02bfc9320119b051f378449cf5f56a1273597a3d99")
 
     depends_on("py-setuptools@40.8:", type="build")
     depends_on("cmake@3.18:", type="build")
     depends_on("py-filelock", type=("build", "run"))
     depends_on("zlib-api", type="link")
 
+    def setup_build_environment(self, env):
+        """Set environment variables used to control the build"""
+        if "%clang" in self.spec:
+            env.set("TRITON_BUILD_WITH_CLANG_LLD", "True")
```

Comment:
```suggestion
        if self.spec.satisfies("%clang"):
            env.set("TRITON_BUILD_WITH_CLANG_LLD", "True")
```

Recommendation to use `spec.satisfies()` instead of `in`.

Reply:
thanks! forgot about that :)

Reply:
No problem, we've got a bunch of the older examples lying around in the repo we still need to fix up :)